whylabs_toolkit/monitor/diagnoser/converters/granularity.py

```diff
@@ -0,0 +1,53 @@
+from dateutil.relativedelta import relativedelta
+from whylabs_toolkit.utils.granularity import Granularity
+from isodate import parse_datetime, parse_duration, parse_date
+
+
+def batches_to_timedelta(time_period: str, batches: int) -> relativedelta:
+    if time_period == "PT1H":
+        return relativedelta(hours=batches)
+
+    if time_period == "P1W":
+        return relativedelta(weeks=batches)
+
+    if time_period == "P1M":
+        return relativedelta(months=batches)
+
+    return relativedelta(days=batches)
+
+
+def time_period_to_granularity(time_period: str) -> Granularity:
```

Comment:
there is a similar "conversion" happening on the helpers package: https://github.com/whylabs/whylabs-toolkit/blob/mainline/whylabs_toolkit/helpers/monitor_helpers.py#L81 maybe we could refactor this to be part of the toolkit in general?

Reply:
refactored